### PR TITLE
Update nederlandse_spoorwegen.markdown

### PR DESCRIPTION
**Pull request by YarmoM**
**Description:**
The NS API has recently changed and this documentation update reflects the changes made in the component codebase.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30599

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by YarmoM** [comment]
Thanks for the tweak and your time reviewing this PR!

File: source/_integrations/nederlandse_spoorwegen.markdown [hunk 1]
```diff
@@ -10,16 +10,15 @@ ha_release: 0.57
...
-Add the data to your `configuration.yaml` file as shown in the example:
+The `nederlandse_spoorwegen` integration can be configured using `configuration.yaml` as shown below:
 
 ```yaml
 # Example configuration.yaml entry
 sensor:
 - platform: nederlandse_spoorwegen
-  email: you@example.com
-  password: !secret ns_password
+  aki_key: NS_API_KEY
```
Comment (amelchio) [inline source/_integrations/nederlandse_spoorwegen.markdown]: Typo.
```suggestion
  api_key: NS_API_KEY
```

Comment (YarmoM) [reply source/_integrations/nederlandse_spoorwegen.markdown]: Good catch!

**Review by frenck — APPROVED** [review]

**Comment by kevintuhumury** [comment]
@YarmoM the API you mentioned in this PR might be outdated before this gets merged into master. You mentioned that the `RetrieveTripInformationPublicAPI` API should be used. Looking at the corresponding page it mentions the following:

> Per the 31st of january only the price API will be available here. Please go to the Ns app Product on this website for using the other API's that are available. Only the Price API will work after the 31st of january!

Source: https://apiportal.ns.nl/products/PublicNsApi

So, I'm not sure whether or not this is going to work after that date. Just thought I'd mention this.

**Comment by YarmoM** [comment]
@kevintuhumury Thanks a bunch for mentioning this! I hadn't realized this change was about to happen.

So for my understanding, I need to point to "NS app" product and then "Reisinformatie" API?

**Comment by kevintuhumury** [comment]
@YarmoM I'm not completely sure. I haven't looked into the API's themselves. I was going through your updated documentation as I want to use the integration myself. So, I went to the API portal and noticed the mention of them closing the API.

It does look like you should use the 'NS app' product, but I'm unsure whether the 'Reisinformatie' API provides the same or more data.

**Pull request merged**